decidim-core/lib/decidim/core/test/shared_examples/comments_examples.rb

```diff
@@ -1039,9 +1039,48 @@
 
       it "shows a message indicating that comments are disabled" do
         visit resource_path
-        expect(page).to have_content("Comments are disabled at this time")
+        expect(page).to have_content("Comments are currently disabled, only administrators can reply or post new ones.")
         expect(page).to have_no_content("You need to be verified to comment at this moment")
       end
+
+      shared_examples "can answer comments" do
+        it "can answer" do
+          visit resource_path
+          expect(page).to have_link("Comment")
+          page.find("a", text: "Comment").click
+          fill_in "Comment", with: "Test admin commenting in a closed comment."
+          click_on "Publish comment"
+          expect(page).to have_content("Test admin commenting in a closed comment.")
+
+          expect(page).to have_button("Reply")
+          first("button", text: "Reply").click
+          expect(page).to have_css(".comment-thread")
+          within first(".comment-thread") do
+            fill_in "Comment", with: "Test admin replying a closed comment."
+            click_on "Publish reply"
+          end
```

Comment:
If here a callout is generated with a "successfully comented" kind of comment, we should check it

Reply:
No, there isn't.